libsession/src/main/java/org/session/libsession/messaging/sending_receiving/ReceivedMessageHandler.kt

```diff
@@ -294,6 +289,13 @@ private fun handleNewClosedGroup(sender: String, sentTimestamp: Long, groupPubli
     storage.setExpirationTimer(groupID, expireTimer)
     // Notify the PN server
     PushNotificationAPI.performOperation(PushNotificationAPI.ClosedGroupOperation.Subscribe, groupPublicKey, storage.getUserPublicKey()!!)
+    // Notify the user
+    if (userPublicKey == sender) {
```

Comment:
At first glance this doesn't seem like something we want to do. If you get added to a closed group you only want to show the "Closed group created" message if it's actually new. Can you explain the rationale behind this change?

Reply:
The message displayed here is 'XXX added you to the group', I thought it would be good to display that message when the user is re-added as well.

Reply:
Alright, that makes sense :)